### PR TITLE
[SCB-518] ServiceCenter ip address need to config default port

### PR DESCRIPTION
**Pull request by weichao666**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---
for http/https, the default ports are 80 and 443. if port is not configured, the default port needs to be used. this simplifies the configuration.

### CODE REVIEW

**Comment by weichao666** [comment]
image

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java [hunk 1]
```diff
@@ -133,10 +131,8 @@ public boolean isSsl() {
     uriList.forEach(anUriList -> {
       try {
         URI uri = new URI(anUriList);
-        StringBuilder sb = new StringBuilder(uri.getHost());
-        sb.append(':').append(uri.getPort() < 0 ? PROTOCOL_HTTP_PORT : uri.getPort());
         this.ssl = uri.getScheme().startsWith("https");
-        ipPortList.add(NetUtils.parseIpPort(sb.toString()));
+        ipPortList.add(NetUtils.parseIpPort(uri.getScheme(), uri.getAuthority()));
```
Comment (liubao68) [inline service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java]: Can you add a UT for getIpPort and https schema?

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java [hunk 2]
```diff
@@ -133,10 +131,8 @@ public boolean isSsl() {
     uriList.forEach(anUriList -> {
       try {
         URI uri = new URI(anUriList);
-        StringBuilder sb = new StringBuilder(uri.getHost());
-        sb.append(':').append(uri.getPort() < 0 ? PROTOCOL_HTTP_PORT : uri.getPort());
         this.ssl = uri.getScheme().startsWith("https");
```
Comment (liubao68) [inline service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java]: I think it's better use "https" equals uri.getScheme

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java (hunk 1 above)
Comment (weichao666) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java]: done

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java (hunk 2 above)
Comment (weichao666) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java]: done

File: service-registry/src/test/java/org/apache/servicecomb/serviceregistry/config/TestServiceRegistryConfig.java [hunk 3]
```diff
@@ -30,6 +34,7 @@
   @BeforeClass
   public static void initClass() {
     ArchaiusUtils.resetConfig();
+    ConfigUtil.installDynamicConfig();
```
Comment (wujimin) [inline service-registry/src/test/java/org/apache/servicecomb/serviceregistry/config/TestServiceRegistryConfig.java]: so heavy, why not just invoke ArchaiusUtils.setProperty inside test case.

Comment (weichao666) [reply service-registry/src/test/java/org/apache/servicecomb/serviceregistry/config/TestServiceRegistryConfig.java]: done

**Pull request merged**